### DISCUSSION TITLE
Welcome to REC Discussions!

### DISCUSSION THREAD

**Discussion by hammar** [RealEstateCore/rec#114] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:

  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sweco-secfbr** [comment]
Hi, Is there a way to declare a rotation of an device in REC?

**Reply by hammar** [reply]
Could you clarify what you are looking for? E.g., physical rotation of the device/it's sensing apparatus/coverage area, or rotation in some other sense?
